lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -205,6 +210,15 @@ def test_facebook_url(self):
         course_data = response.data[0]['course']
         self.assertEquals(course_data['social_urls']['facebook'], self.course.facebook_url)
 
+    @patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
+    def test_discussion_url(self):
+        expected_discussion_url = 'http://testserver/api/discussion/v1/courses/%s' % self.course.id
+        self.login_and_enroll()
+
+        response = self.api_response()
+        course_data = response.data[0]['course']
+        self.assertEqual(expected_discussion_url, course_data['discussion_url'])
```

Comment:
Let's not make this test dependent on the test server's domain name.
Can we change this line to:
`self.assertIn('/api/discussion/', course_data['discussion_url'])`
or
`self.assertIn(`/api/discussion/v1/courses/{}'.format(self.course.id), course_data['discussion_url'])`
if you want to include the course_id.